02-Reproject-Raster-In-R.Rmd

```diff
@@ -194,32 +205,34 @@ crs(DTM_hill_HARV)
 extent(DTM_hill_UTMZ18N_HARV)
 extent(DTM_hill_HARV)
 
-
 ```
 
 Notice in the output above that the CRS of `DTM_hillUTMZ18N_HARV` is now UTM. 
 However, the extent of the object is different. 
+
 Why do you think this is?  
 
+Note: When you are reprojecting a raster, you are moving it from one "grid" 
```

Comment:
I think we need a div tag potentially.
 
But also i'm noticing you don't like notice1. I actually like it visually.
But i don't see a good reason to have two styles at this point. Let's talk more and then we can implement throughout! 

Issue: 
https://github.com/NEONInc/NEONDS-Template-Repository/issues/6